static/js/util/profile_edit.js

```diff
@@ -131,15 +123,8 @@ export function boundTextField(keySet, label) {
  * bind this to this.boundSelectField in the constructor of a form component
  * to update select fields
  * pass in the name (used as placeholder), key for profile, and the options.
- *
- * @param keySet {String[]} Path to the field
- * @param label {String} Label for the field
- * @param options {Object[]} A list of options for the select field
- * @param onChange {func} Handler which is called when the state changes or is cleared.
- * Takes the updated profile as argument
- * @returns {ReactElement}
  */
-export function boundSelectField(keySet, label, options, onChange) {
+export function boundSelectField(keySet: string[], label: string, options: Object[], onChange: Function|void) {
```

Comment:
Can you specify a type for `onChange`? It should take only a profile as its argument